WTWSMS/history/titles/k_mimamsa.txt

```diff
@@ -0,0 +1,24 @@
+20.1.1={
+	holder= 0
+	active = yes
+}
+
+476.2.2 = {
+	holder = 0
+}
+
+480.5.1 = {
+	holder = 0
+}
+
+480.12.10 = {
+	holder = 0
+}
+
+486.1.1 = {
+	holder = 0
+}
+
+511.11.28 = {
+	holder = 0
+}
```

Comment:
@loup99 

I thought Character Zero means "generate a random character". What does it actually mean?